rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -615,7 +616,7 @@ void cell_audio_thread::advance(u64 timestamp)
 	for (u32 i = 0; i < queue_count; i++)
 	{
 		lv2_obj::notify_all_t notify;
-		queues[i]->send(event_sources[i], 0, 0, 0);
+		queues[i]->send(event_sources[i], CELL_AUDIO_EVENT_MIX, 0, event_data3[i]);
```

Comment:
iirc when I tested on realhw these values were indeed 0. Iirc even documentation suggests they are unused and should not be read.

Reply:
CELL_AUDIO_EVENT_MIX is 0.
Data 3 is also 0 unless you use the before mix flag. It is used to acknowledge the audio mix for that queue